frontend/src/component/Search/DateFilter.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import DayPickerInput from 'react-day-picker/DayPickerInput'
+import 'react-day-picker/lib/style.css'
+import dateFnsFormat from 'date-fns/format'
+import { DATE_FILTER_ELEMENT } from '../../util/helper.js'
+
+export class DateFilter extends React.Component {
+
+  formatDate = (date, format, locale) => {
+    return dateFnsFormat(date, format, { locale })
+  }
+
+  render() {
+    const { props } = this
+
+    const WEEKDAYS_LONG = {
```

Comment:
I know this is WIP but I don't think you should have to create every translations for week day and months
Firstly because, if we add a language, we would have to update the code and secondly, your date library must handle its translations

Reply:
I added a comment and this issue https://github.com/tracim/tracim/issues/4212